spec/creditsafe/client_spec.rb

```diff
@@ -8,8 +8,8 @@
 
 RSpec.describe(Creditsafe::Client) do
   notifications = []
-  let(:username) { "b" }
-  let(:password) { "c" }
+  let(:username) { "bob" }
+  let(:password) { "hunter2" }
```

Comment:
if we want [to go full on idiomatic-meme](http://bredsaal.dk/top-five-bash-org-quotes), the username should be `AzureDiamond`.